[28기 윤종호]로그인 & 메인페이지 필수 구현 완료

sole77850 opened the pull request:
- 로그인 페이지 완성
- 메인 페이지 레이아웃만 잡음
- 단순 section 구분만 완성

## :: 최근 작업 주제 (하나 이상의 주제를 선택해주세요.)
- [x] 기능 추가
- [ ] 리뷰 반영
- [x] 리팩토링
- [ ] 버그 수정
- [ ] 컨벤션 수정

<br />

## :: 구현 목표 (해당 브랜치에서 구현하고자 하는 하나의 목표를 설정합니다.)
- 로그인 기능 구현
- westagram 메인 페이지 기본 레이아웃 잡기

<br />

## :: 구현 사항 설명 (작업한 내용을 상세하게 기록합니다.)
1. 로그인 페이지 구현
- 입력창에 변화가 일어날 때 마다 로그인 버튼 활성화 조건 검토
- 로그인 페이지 레이아웃 완성

2. Westagram 메인 페이지 기본 레이아웃
- 메인 페이지 기본 레이아웃 잡기
- Section 태그를 통한 전체 영역 구분만 완성
- 세부 이미지 태그 삽입

<br />

## :: 성장 포인트 (해당 기능을 구현하며 고민했던 사항이나 새로 알게된 부분, 어려웠던 점 등을 작성합니다.)
- html/css를 이용한 레이아웃 작성 시, 각 요소들의 기본적인 성질 및 정의를 분명히 함.
- 코드의 간결함을 지속해서 고려

<br />

## :: 기타 질문 및 특이 사항
- 웹 페이지 레이아웃 작성 시, column 형태를 구성할 때 grid/flex/position 등의 방법 중 최근의 사용 경향은 어떤지 궁금합니다.

joonsikyang commented on students/14th/hyesoo-ahn/insta.js:
```diff
@@ -0,0 +1,53 @@
+// id 1 이상, pw 6 이상  document.que('').value;
+// querySelector('. #')로 선택해서 color keyup했을때 변경 
+
+const Name = document.getElementById('name');
+const Pw = document.getElementById('pw');
+const Btn = document.getElementById('btn');
```
- 변수가 대문자로 시작하는 이유가 있나요? camelCase로 맞춰주세요 :)
- 불필요한 주석은 전부 삭제해주세요.

joonsikyang commented on students/14th/hyesoo-ahn/insta.js:
```diff
@@ -0,0 +1,53 @@
+//     }
+// })
+
+document.addEventListener('keyup', function () {
+    Btn.style.backgroundColor = Name.value && Pw.value ? '#0095F6' : '#B7DFFC';
+})
```
- 삼항 연산자 활용 Goood 🙌
- document 전체에 이벤트를 걸어줘도 동작하겠지만 좀 더 구체적인 요소에 이벤트를 주는게 좋을 것 같습니다. Name 요소와 Pw 요소를 감싸는 요소에 이벤트를 부여해보세요!

joonsikyang commented on students/14th/hyesoo-ahn/insta.js:
```diff
@@ -0,0 +1,53 @@
+document.addEventListener('keyup', function () {
+    Btn.style.backgroundColor = Name.value && Pw.value ? '#0095F6' : '#B7DFFC';
+})
+// Name과 Pw에서 각각 addEventListener 를 적용했는데, 2개의 input 창에서 모두 동작해야 하므로, document 자체에 addEventListener 걸기!⇒ document 전체를 추적하다가 이벤트 조건에 맞으면(Name.value && Pw.value) 이벤트 실행
+// 삼항연산자 
```
주석 Good! 팀 프로젝트 시에는 개인 메모장에 작성해주세요🤭

hyesoo-ahn replied:
오아우 멘토님... 자세한 피드백 너무 감사합니다 !!

joonsikyang reviewed (commented):
혜수님 수고하셨습니다 :) 리뷰 내용 확인하고 반영해주세요!

Commit:
[Add]로그인 페이지 완성 & 메인페이지 레이아웃
- 로그인 페이지 완성
- 메인 페이지 레이아웃만 잡음
- 단순 section 구분만 완성

Commit:
[28기 윤종호]메인 페이지 레이아웃 1차 완료
- 단순 레이아웃 구성 완료
- class, id 등수정하며 html, css 코드 단순화 작업 예정
- 코드 단순화 이후 Westagram wrap에서 배운 내용 반영
- form, 함수 단순화 etc.

Commit:
[28기 윤종호]reset.css, common.css 첫 작성
- html, body 영역 reset
- 몇가지 공통 속성 값을  사용자 지정 속성으로 대체

Commit:
[28기 윤종호] navbar 레이아웃 코드 변경
- 웹페이지 스크롤 시에도 상단에 navbar 위치하도록 position: fixed
  속성으로 변경
- nav bar 관련 CSS 전체 삭제 및 새로 작성
- 디자인 관련 세부 작업은 미완료, 현재 진행중

Commit:
[28기 윤종호]nav bar 디자인 수정
- 각 이미지 크기 조정
- 요소 위치 수정

Commit:
[디자인수정]
- font size 등 단순 수정

Commit:
[필수구현완료] Westagram 구현 완료
- 로그인, 메인페이지 레이아웃 완료
- 댓글 추가 기능 완료
- 필수 요소 완료
